docs/trigger-guide.md

```diff
@@ -49,6 +48,36 @@ The framework provides support to fetch trigger resources from different sources
 - [Gateway](gateway-protocol.md)
 - [Sensor](sensor-protocol.md)
 
+  * ### Trigger Standard Kubernetes Resource
+    There could be a case where you may want to trigger a standard Kubernetes resource like Pod, Deployment etc. instead of an Argo Workflow.
+    The sensor allows you to trigger any K8s resource in the same way you would trigger an Argo Workflow.
+
+    * The [example](../examples/sensors/trigger-standard-k8s-resource.yaml) showcases how you can trigger different standard K8s resources.
+      You can find K8s API reference [here](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.13/).
+
+    * To trigger other standard K8s resources, change the group and version in `triggers/resource` accordingly.
+
+  * ### Trigger Gateway
+    Follow the [example](../examples/sensors/trigger-gateway.yaml) to trigger a gateway. 
+
+    In the example, the sensor creates a configmap and gateway resource for artifact gateway.
+ 
+    Because a gateway depends on `gateway-configmap` which stores the event source configurations, the first trigger in sensor is a configmap
```

Comment:
This sentence isn’t entirely clear .. what does “first trigger in sensor is a configmap”?

Reply:
Ahh nvm this makes sense now